genesyscloud/group/resource_genesyscloud_group_test.go

```diff
@@ -54,7 +55,7 @@ func TestAccResourceGroupBasic(t *testing.T) {
 			},
 			{
 				// Update group
-				Config: GenerateUserWithCustomAttrs(testUserResource, testUserEmail, testUserName) + generateGroupResource(
+				Config: generateUserWithCustomAttrs(testUserResource, testUserEmail, testUserName) + GenerateGroupResource(
```

Comment:
I had to duplicate functions to break cyclic dependencies.  As you look through the code you mind find duplicate functions across packages.  We will need to clean this up at some point and centralize into a single package, but for now I am just moving functions locally.